backend/Squirrel.Shared/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Net;
+using Squirrel.Shared.Enums;
+using Squirrel.Shared.Exceptions.Abstract;
+
+namespace Squirrel.Shared.Exceptions;
+
+public sealed class NotFoundException : RequestException
```

Comment:
Maybe rename to `EntityNotFoundException`, because you throw it only when entity is not found

Reply:
updated